### DISCUSSION TITLE
DT-DX MiniDexed hardware by Dtronics

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#637] [Show and tell]
Designed by Dtronics
Inspired by https://github.com/probonopd/MiniDexed/

Sneak preview:
https://www.youtube.com/watch?v=NZfg9YMF5rw

![image](https://github.com/probonopd/MiniDexed/assets/2480569/83e51a0b-2135-4cd3-a253-f99cbe10cf39)

While MiniDexed has been designed so that you can build do-it-yourself gear using inexpensive and widely available parts, it is great to see that there is now a readymade, studio-level rugged hardware option available.

Details:
https://www.dtronics.nl/dt-dx

(I am not affiliated with Dtronics and do not financially benefit from their device sales.)

**Comment by plamikcho** [comment]
Unfortunately you cannot buy it yet - the link is opening another product but probably they will fix it.

**Comment by probonopd** [comment]
Example configuration - rename to `minidexed.ini`:

[minidexed.txt](https://github.com/probonopd/MiniDexed/files/14968668/minidexed.txt)

Note that the DT-DX uses a different pin configuration than the standard ini file.

**Comment by Banana71** [comment]
Amazona.de also reports
https://www.amazona.de/dtronics-dt-dx-fm-synthesizer-auf-minidexed-basis/

**Comment by diyelectromusic** [comment]
> (I am not affiliated with Dtronics

I'm not sure their website got the message :)

> "The Dtronics DX was created in collaboration with Probonopd. He manages the MiniDexed project on Github."

Kevin

**Reply by probonopd** [reply]
Well, the device uses our firmware, and I did give feedback on a prototype (as I would give anyone interested in it). But I don't have, and am not looking for, a business relationship.

**Comment by soyersoyer** [comment]
Kickstarter campaign:
https://www.kickstarter.com/projects/dtronics/dtronics-dt-dx/

**Reply by diyelectromusic** [reply]
Wow.  That's quite a mark-up for what is essentially something [like this](https://diyelectromusic.com/2022/07/28/minidexed-raspberry-pi-io-board/) in a box... especially considering "the MiniDexed software is included free of charge" to quote the page...

Still, good job they contacted /the/ developer...

More seriously, I'm a little nervous about this claim: "to be used as a professional musical instrument on stage, in the studio and at home".  That might apply to their hardware - I don't know, I've not seen any designs - but I'd not make that claim about the firmware.  We don't even have a proper release channel...

Kevin

**Reply by Banana71** [reply]
> That's quite a mark-up for what is essentially something [like this](https://diyelectromusic.com/2022/07/28/minidexed-raspberry-pi-io-board/) in a box...

Yes, but the LCD now glows orange.  :jack_o_lantern:

I should put the "Rockstar" performance back into the project, or the Dtronics customers won’t find the advertised sounds on their devices. :gift:

**Reply by Lonolo** [reply]
Is there external access to the SD card?

**Reply by probonopd** [reply]
No, in order to access the microSD card one has to open the housing and remove the bottom cover.
But maybe with a future MiniDexed version that has integrated networking, there will be less need to access the microSD card physically.
